### ISSUE TITLE
[Suggestion] File select screen for Add WADs Menus in Setup

### ISSUE THREAD

**Issue by OpenRift412** [chocolate-doom/chocolate-doom#1437]
Could we perhaps get a button in the Add WADs Menus that opens a file select window that allows us to more easily select a WAD? Typing out absolute or relative paths is oftentimes tedious, and it doesn't help if something goes wrong and everyone has to type _everything_ back in when they start up the setup program again.

**Comment by fragglet** [comment]
Indeed, yes. The code to do this actually already exists but I had to disable it in c5defc73afb3cef52ab26 because it was too buggy. It should really be fixed and re-enabled.

**Comment by AXDOOMER** [comment]
The commit that disabled this feature on Windows is very old. Anyone knows if this would still be an issue on Windows 10?

**Comment by atsb** [comment]
To be honest, SDL2 does not work on Windows XP and Windows Vista is deprecated.  Should we really consider users who are using 20+ and almost 20 year old operating systems?

I will re-enable this for Sprinkled Doom and see how it works on Windows 10.

**Comment by atsb** [comment]
Well there is a bug..

```
static int LoadDLLs(void)
{
    HMODULE comdlg32, shell32 <-- no semicolon
```

I guess someone forgot to add that because it was never enabled code.  But the dialog box on Windows 10 opens, closes, selects wads etc just fine and does not crashes.

1. WADS do NOT appear in the dialog, you need to type them into the select menu at the bottom.
2. WADS are not saved once you hit F10 and once you go to the setup program again, the list is empty.
